contrib/completion/zsh/_docker-compose

```diff
@@ -359,10 +362,17 @@ _docker-compose() {
 
     _arguments -C \
         '(- :)'{-h,--help}'[Get help]' \
-        '--verbose[Show more output]' \
-        '(- :)'{-v,--version}'[Print version and exit]' \
         '(-f --file)'{-f,--file}'[Specify an alternate docker-compose file (default: docker-compose.yml)]:file:_files -g "*.yml"' \
         '(-p --project-name)'{-p,--project-name}'[Specify an alternate project name (default: directory name)]:project name:' \
+        '--verbose[Show more output]' \
+        '(- :)'{-v,--version}'[Print version and exit]' \
+        '(-H --host)'{-H,--host}'[Daemon socket to connect to]:host:' \
```

Comment:
Since you are alphabetically re-ordering `--verbose` and `-v --version`, can you do the same with the new options (`--tls` etc.) ?
I would also prefer to have separated commits per group of new command options, for a better changes history lisibility:
- One for `docker-compose` base options (`--tls` etc.)
- One for adding `--remove-orphans` to `up` and `down`.
- One making `--force-recreate`, `--no-recreate`, `--no-build` and `--build` common between `create ` and `up` commands. See my other comment.
- One for fixing little errors.